samples/localization/SharedProject/LocalizationTest/LocalizationTest/LocalizationTest/Services/Localizer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Reflection;
+using System.Resources;
+using System.Text;
+
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(LocalizationTest.Services.Localizer))]
+namespace LocalizationTest.Services
+{
+    public class Localizer : ILocalizer
+    {
+        
```

Comment:
Remove blank line in the template

Reply:
I'm going to remove this particular sample solution (shared project) as it was added for the Xamarin team (David and Pierce) to provide some feedback on.

Reply:
This one isn't part of the template.  The .NetStd version of the sample project is functional and can be used to demonstrate how to use the capability in addition to an example page template.